include/triton/common/triton_json.h

```diff
@@ -239,6 +239,15 @@ class TritonJson {
       return TRITONJSON_STATUSSUCCESS;
     }
 
+    // Set/overwrite an unsigned integer in a value. This changes the
+    // type of the value to unsigned int.
+    TRITONJSON_STATUSTYPE SetUInt(const uint64_t value)
+    {
+      rapidjson::Value& v = AsMutableValue();
+      v.SetUint64(value);
+      return TRITONJSON_STATUSSUCCESS;
+    }
+
```

Comment:
1. Is a SetDouble needed to go with `double_param` addition?
2. For the `// FIXME` comment a few lines up, maybe it can be removed?

Reply:
I've added the SetDouble - though I don't believe we use it anywhere at the moment. I've removed the comment - although we don't mirror all "set" types - I don't think we actually need to so think its safe to expand as we need to.